judith_shimer/midterm/lib/seeds.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'shelter'
+require_relative 'animal'
+require_relative 'client'
+
+@happitails = Shelter.new('HappiTails')
```

Comment:
happitails doesn't need to be an instance variable since happitails itself represents an instance of the shelter class.